.github/workflows/main.yaml

```diff
@@ -244,7 +244,7 @@ jobs:
         with:
           test_targets: ${{ needs.build.outputs.test_targets }}
           test_results_key: ${{ env.TEST_RESULTS_KEY }}
-          gcs_results_path: gs://cobalt-unittest-storage/results/${{ matrix.name }}/${{ github.run_id }}
+          gcs_results_path: gs://cobalt-unittest-storage/results/${{ matrix.name }}/${{ github.run_id }}/${{ github.run_attempt }}
```

Comment:
This doesn't use job id as the title of the pull request implies.

Reply:
fixed